webpack.config.babel.js

```diff
@@ -93,17 +101,50 @@ const webPackConfig = {
     path: path.resolve(__dirname, 'dist'),
     publicPath: '/',
   },
+  devServer: {
+    hot: true,
+    index: '',
+    proxy: [
+      {
+        // Proxy all server-served routes:
+        context: ['/img', '/api', '/auth', '/socket.io', 'sockjs-node'],
+        target: 'http://localhost:9090',
+      },
+      {
+        context: ['/socket.io', 'sockjs-node'],
+        target: 'ws://localhost:9090',
+        ws: true,
+      },
+      // Proxy root to server to mirror the server routes (goes to /app/home currently)
+      {
+        context: '/',
+        target: 'http://localhost:9090',
+        // eslint-disable-next-line consistent-return
+        bypass(req) {
+          return req.path !== '/' ? '/index.html' : undefined
+        },
+      },
+    ],
+    compress: false,
+    port: 9000,
+    historyApiFallback: true,
+  },
   module: {
     rules: [
       {
         test: /\.(js|jsx)$/,
-        exclude: /(node_modules|bower_components)/,
-        use: {
-          loader: 'babel-loader',
-        },
+        exclude: /node_modules/,
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              plugins: isDevelopment ? [require.resolve('react-refresh/babel')] : [],
+            },
+          },
+        ],
       },
       {
-        test: /\.(sa|sc|c)ss$/,
+        test: /\.(sass|scss|css)$/,
```

Comment:
was this change necessary ?

Reply:
I don't know if it's necessary or not but to me it looks more clear in this way

Reply:
For me too 👍

Reply:
For grepping